example/gorilla/math/client/generated_MathClient.go

```diff
@@ -34,7 +46,7 @@ func (c *Math) IdentityMany(args []int) (*[]int, error) {
 	return reply, err
 }
 
-func (c *Math) IdentityManyStruct(args []math.IdentityStruct) (*[]math.IdentityStruct, error) {
+func (c *Math) IdentityManyStruct(args []*math.IdentityStruct) (*[]math.IdentityStruct, error) {
```

Comment:
Do you get any value by passing all input and output by pointers?

Reply:
Weird, it should be dereferencing... but I don't think it's related to this PR. I think master just wasn't go gen'd before. IdentityMany() also has *[]. I'll take a look soon separately